packages/excalidraw/scene/Scene.ts

```diff
@@ -229,12 +234,25 @@ class Scene {
 
   replaceAllElements(
     nextElements: readonly ExcalidrawElement[],
-    mapElementIds = true,
+    mapOfIndicesToFix?: Map<string, ExcalidrawElement>,
   ) {
-    this.elements = nextElements;
+    let _nextElements;
+    if (mapOfIndicesToFix) {
+      _nextElements = fixFractionalIndices(nextElements, mapOfIndicesToFix);
```

Comment:
@dwelle should double-check if we are passing this param on all the necessary places